File: Microsoft .NET/ASP.NET/VB.NET MVC Web Reporting Sample/MVC Web Reporting Sample/ViewModels/StartPageModel.vb
Imports System.Collections.Generic

Namespace WebReporting.ViewModels
    Public Class StartPageModel

        ''' <summary>
        ''' List of items of the custom repository type defined for this sample
        ''' </summary>
        Public Property RepositoryItems() As IEnumerable(Of CustomizedRepostoryItem)
            Get
                Return m_RepositoryItems
            End Get
            Set
                m_RepositoryItems = Value
            End Set
        End Property
        Private m_RepositoryItems As IEnumerable(Of CustomizedRepostoryItem)
    End Class
End Namespace

File: Microsoft .NET/.NET Core 3.1/ASP.NET/C#/MVC Web Reporting Sample/MVC Web Reporting Sample/ViewModels/StartPageModel.cs
using System.Collections.Generic;

namespace WebReporting.ViewModels
{
    public class StartPageModel
    {

        /// <summary>
        /// List of items of the custom repository type defined for this sample
        /// </summary>
        public IEnumerable<CustomizedRepostoryItem> RepositoryItems { get; set; }
    }
}